plugins/trader/portfolioManager.js

```diff
@@ -344,10 +344,10 @@ Manager.prototype.relayOrder = function(done) {
 
     var price = 0;
     var amount = 0;
-    var date = moment(0);
+    var date = moment().unix();
 
     _.each(res.filter(o => !_.isUndefined(o) && o.amount), order => {
-      date = _.max([moment(order.date), date]);
+      date = _.max([moment(order.date).unix(), date]);
```

Comment:
this broke live all gekkos, see #1862.

Reply:
Can you explain why you changed this? For now I am reverting to the old behaviour.